apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -1,42 +1,333 @@
+                      flex: '1 1 auto',
+                    },
+                    wrapper: {
+                      background: 'transparent',
+                      width: '100%',
+                    },
+                    input: {
+                      background: 'transparent',
+                      borderStyle: 'solid',
+                      borderColor: colorScheme === 'dark' ? theme.colors.dark[5] : theme.colors.gray[5],
+                      borderWidth: '1px',
+                      fontSize: '20px',
+                      fontWeight: 'bolder',
+                      padding: 9,
+                      lineHeight: '28px',
+                      minHeight: 'auto',
+                      height: 'auto',
+                      width: '100%',
+                      textOverflow: 'ellipsis',
+                      '&:not(:placeholder-shown)': {
+                        borderStyle: 'none',
+                        padding: 10,
+                      },
+                      '&:hover, &:focus': {
+                        borderStyle: 'solid',
+                        padding: 9,
+                      },
+                      '&:disabled': {
+                        backgroundColor: colorScheme === 'dark' ? colors.B15 : theme.white,
+                        color: colorScheme === 'dark' ? theme.white : theme.black,
+                        opacity: 1,
+                      },
+                    },
+                  })}
+                  {...field}
+                  value={field.value !== undefined ? field.value : selectedProvider.displayName}
+                  // error={showErrors && fieldState.error?.message}
+                  type="text"
+                  data-test-id="title"
+                  placeholder="Enter workflow name"
+                  // disabled={readonly}
+                />
+              );
+            }}
+          />
+          <ActionIcon
+            variant={'transparent'}
+            onClick={() => {
+              navigate('/integrations');
+            }}
+          >
+            <Close color={colors.B40} />
+          </ActionIcon>
+        </Group>
+        <Group mb={16} spacing={16}>
+          <IntegrationChannel name={CHANNEL_TYPE_TO_STRING[selectedProvider.channel]} type={selectedProvider.channel} />
+          <IntegrationEnvironmentPill
+            name={
+              environments?.find((environment) => environment._id === selectedProvider.environmentId)?.name ||
+              'Development'
+            }
+          />
+        </Group>
+        <CenterDiv>
+          <ActiveWrapper active={isActive}>
+            <Controller
+              control={control}
+              name="active"
+              render={({ field }) => <Switch checked={isActive} data-test-id="is_active_id" {...field} />}
+            />
+            <StyledText data-test-id="connect-integration-form-active-text">
+              {isActive ? 'Active' : 'Disabled'}
+            </StyledText>
```

Comment:
```suggestion
          <ActiveWrapper active={isActive}>
            <Controller
              control={control}
              name="active"
              render={({ field }) => <Switch checked={isActive} label= {isActive ? 'Active' : 'Disabled'} data-test-id="is_active_id" {...field} />}
            />
         
```
According to the design, the Active should be gradient. What I suggested here will do the job :)